src/compiler/diagnosticMessages.json

```diff
@@ -2513,7 +2513,7 @@
         "category": "Message",
         "code": 6019
     },
-    "Compile the project in the given directory.": {
+    "Compile the project in the given directory, using 'tsconfig.json' or the specified config file.": {

```

Comment:
Compile a project given the path to its configuration file, or to a folder with a 'tsconfig.json'.